### ISSUE TITLE
Accessibility issue: Images must have alternative text on /notebook-ci-testing/notebooks/mosaic_pipeline/mosaic_pipeline.html

### ISSUE THREAD

**Issue by dulude** [spacetelescope/notebook-ci-testing#348]
## What
An accessibility scan flagged an element on https://spacetelescope.github.io/notebook-ci-testing/notebooks/mosaic_pipeline/mosaic_pipeline.html because images must have alternative text. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/image-alt?application=playwright.

The following element needs attention:

- `<img src="https://raw.githubusercontent.com/spacetelescope/roman_notebooks/refs/heads/main/stsci_logo2.png" width="90">` (selector: `img[width="90"]`)

To fix this, ensure <img> elements have alternative text or a role of none or presentation.

Specifically:

Fix any of the following:
-   Element does not have an alt attribute
-   aria-label attribute does not exist or is empty
-   aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
-   Element has no title attribute
-   Element&apos;s default semantics were not overridden with role="none" or role="presentation"

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.2 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
